doc/man1/flux-resource.rst

```diff
@@ -102,11 +102,14 @@ COMMANDS
 OUTPUT FORMAT
 =============
 
-The *--format* option can be used to specify an output format using Python's
-string format syntax or a defined format by name. For a list of built-in
-and configured formats use ``-o help``. See :man1:`flux-jobs` *OUTPUT FORMAT*
+The *--format* option can be used to specify an output format using
```

Comment:
I'm assuming it was accidental that this was formatted to ~70 columns width instead of 78 like the rest of the file.